crates/sui-core/src/transaction_manager.rs

```diff
@@ -289,6 +289,7 @@ impl Inner {
         &mut self,
         input_key: InputKey,
         update_cache: bool,
+        metrics: Arc<AuthorityMetrics>,
```

Comment:
Taking metric by ref should be good enough?

Reply:
agreed